test/v2/private.read.test.ts

```diff
@@ -0,0 +1,171 @@
+import { RestClientV2 } from '../../src/rest-client-v2';
+import {
+    errorResponseObjectV3,
+    sucessEmptyResponseObject,
+} from '../response.util';
+
+describe('Bitget Private REST API Read Endpoints', () => {
+  const API_KEY = process.env.API_KEY_COM;
+  const API_SECRET = process.env.API_SECRET_COM;
+  const API_PASSPHRASE = process.env.API_PASS_COM;
+
+  const api = new RestClientV2({
+    apiKey: API_KEY!,
+    apiSecret: API_SECRET!,
+    apiPass: API_PASSPHRASE!,
+  });
+
+  it('should have api credentials to use', () => {
+    expect(API_KEY).toStrictEqual(expect.any(String));
+    expect(API_SECRET).toStrictEqual(expect.any(String));
+    expect(API_PASSPHRASE).toStrictEqual(expect.any(String));
+  });
+
+  describe('Account Endpoints', () => {
+    it('getSpotAccount()', async () => {
+      try {
+        const res = await api.getSpotAccount();
+        expect(res).toMatchObject(sucessEmptyResponseObject());
+      } catch (e) {
+        expect(e).toMatchObject(errorResponseObjectV3('40001'));
```

Comment:
From checking the docs, seems 40001 is "ACCESS_KEY cannot be empty". Are we expecting these tests to fail? Seeing an exception with 40001 seems to me like an issue that we want to investigate...either keys are missing or not being processed correctly. Not sure I understood the reasoning here...

if these exceptions shouldn't happen, you can `expect(e).toBeUndefined()` and it'll fail the test if the exception is seen.